src/portfolios/portfolios.controller.ts

```diff
@@ -137,7 +138,7 @@ export class PortfoliosController {
   @Post('/identities/:did/portfolios/:id/delete')
   public async deletePortfolio(
     @Param() portfolio: PortfolioDto,
-    @Query() { signer, webhookUrl }: TransactionBaseDto
+    @Query() { signer, webhookUrl, dryRun }: TransactionBaseDto
```

Comment:
Lint error here, this will cause delete to be removed for real even when dryRun is passed

That said, not sure if the belongs with @Query annotation. This is a weird endpoint due to its history of being under `DELETE` where DELETE isn't supposed to have a body.